### ISSUE TITLE
Is binary bot a scam?

### ISSUE THREAD

**Issue by Bkind254** [binary-com/binary-bot#3570]
Hi, guys what if I have a well-funded (10,000usd) account and you stake as low as 2 dollars, the daily target is 1% per day....will you survive in the market for a long time till you make another 10,000usd  or somewhere along the way you'll blow the account. Can someone trade long-term using the binary bot without blowing the account? Please help with sincere answers.   @hotmatrixx @binary-ex-machina

**Comment by oramaster** [comment]
I believe with a good strategy combined with proper money management; you can achieve that

**Comment by binary-ex-machina** [comment]
### TL;DR is totally possible, but there is a catch, and the majority will never understand (or will want to understand)

Now to a more detailed approach.

@Bkind254, let's check an example based on your 10k account, 100% target, $2 stake, and martingale with factor 2.1

Let's say you have a good strategy, one that gives you 8-10 losses in a row in 500 trades using rise/fall, 10-12 losses in a row in 1000 trades, and 12-14 losses in a row in 2000 trades. Let's assume you have a 50% win rate at the end of the trading cycle.

Rise/fall trade types have that win rate, no matter the indicators or strategy you use. The indicators and money management will shape the number of losses in a row in that period. With a payout of 95% (using synthetic indexes), you have:

• Each winning trade will return $1,9 as the base, no matter how many losses in a row you have
• in 2000 trades, you will have ~ 1000 wins, so ~ $1900 profit (using a fixed stake)
• In 2000 trades, you will have ~ 1000 losses, so  ~ $2000 in losses (using a fixed stake)
• In 2000 trades, you will have ~ 1000 wins and ~ 1000 in losses, so depending on the number of times you get the max number of losses, you will always end in profit (~ $1900 from the winning trades and more from the losing trades, because of martingale)

This gives the idea that you will always end in profit using martingale. But that only will be possible if you have enough balance to make the trades with the stakes necessary to recover from the sequential losses.

You can do the math: Multiply the stake by the factor at the power of the max losses in a row you expect in the trading cycle (2000 trades or more)

$2 (stake) * 2.1 (martingale factor) ^ 14 (max number of losses) = $64.878,39

To recover from 14 losses in a row, you must have left on your account more than $64k. Reducing the stake to 0.35, you will have

$0.35 (stake) * 2.1 (martingale factor) ^ 14 (max number of losses) = $11.353,72

Still, a 10k account can't recover. Let's use the late martingale factor three and apply the martingale only after the third loss in sequence. This will reduce three losses from our maximum expected

$0.35 (stake) * 2.1 (martingale factor) ^ 11 (max number of losses) = $1.225,97

It significantly reduced the amount of capital expected, but it has a catch. Since you will start applying martingale only after the third loss, the amount of profit will be affected. Maybe ending in losses (even if it is small) or a small profit. 

You can use the virtual losses too. However, that will increase the time to reach 100% 3-4 times since you will be skipping wins and losses. This can reduce the number of losses in a row considerably. Still, it's not guaranteed that you will not enter again in another sequence of losses when you skip 3-5 losses in a row. 

Making 1% a day or 100% a day doesn't matter that much on synthetic markets. They are not affected by any external factor (news, open/closing of the market, holidays, etc.). 

No finally, to answer your question

> Can someone trade long-term using the binary bot without blowing the account?
Yes, it's totally possible. So how can we do it?

I will put what I have been using for the last 2 years. Maybe you have perceived this already, but my explanation of the later martingale was very detailed and with some specific numbers. That's because I have used that idea since the day I tested a strategy with it. I really don't like the virtual loss, but some traders have good profits using it. 

I am always considering more than 12 losses in a row on a trading cycle. The trading cycle means the time to achieve my target. I use 6 months period to make 120% as my threshold, which comes first (120% or 6 months trading period). 

I use a significant stop loss (40%), and 2% take profit per session (daily). The strategy generates 360% profit, then measuring the 6 stop losses (total of 240%), I get 120% profit, double in a year. 

Using your $10k account example, that is $24k per year. 

Totally possible, totally achievable, simple to follow. Easy? No. Requires you to trade every day for one year. Requires you to make 2% a day, even if you are sick, traveling, busy, working, or living. Requires you to believe in your system and strategy. And when I write "Believe," I mean you must trust it and be confident about it, know everything about it. Requires you to test 10x more than you usually do for sporadic trading. 

Trading every day to make such a tiny target, knowing that you have a high risk because of the frequency you trade, it's not emotional easier. I tried with any number of systems and bots you can imagine. None of them give you the same adrenaline, sadness, happiness, and emotional feelings that trading manually. 

Binary options are the most challenging trading instrument you will find, it gives you faster profits, and it's addictive. But, like a game, it can suck all your money, the same way it can give you the money to live by it. 

I see binary options as a hedge to other trading instruments. A way to another more complex trading type. It can be the first and the last tool someone uses to trade. But unfortunately, it can ruin trading expectations too. 

Using $50 as capital, or $10.000. If you don't test, manage the trades, dedicate your time (a lot of time, I mean a lot more than you do with your family, friends, job), study, learn, watch about it, repeat, you will never make it. 

You may earn a lot of money fast using a strategy you found online, but I doubt you will remain a trader after some months without blowing your account. Forex, stocks, spot trading, crypto, futures, options, binary options... you can name a lot of trading instruments. Still, if you don't have discipline, consistency, and dedication, you will fail in any of them.

Most of the traders in binary options want fast money, small investment, and significant profit amounts. The Failure rate of those traders is above 95%, so you can understand why it's tough to find someone that makes a living trading only binary options. Besides having the exact mechanics that a casino has, you can trade it like an asset and not gamble. You just need to have the correct perception of it.

**Comment by Bkind254** [comment]
My strategy is as simple as odds and even targeting 0.5% of the account per trade, trading two times a day making it 1% profit. Thanks for this information coz I now believe I can, because am in need of long-term consistency.

**Comment by pugak825** [comment]
After 8 months of testing almost 4,000 bots  ,all bots failed to win in long run.
Seems like they are recording the strategy of your bot the first time you run it and win. Then when you run it again you loss

**Comment by alex-net007** [comment]
![image](https://user-images.githubusercontent.com/61630471/162481603-0cedc5fd-7941-43a9-a093-0959c2fb84cb.png)

Executo meu bot no mt5, são 3 bots 1º. no site da binary 2º. criado no telegram para receber as informações 3º. no Mt5

Desse forma se você acha que o site da binary copia as informações do seu bot ele fica blindado!!

Esse robô da foto trabalha 24hs/7 dias

**Comment by binary-ex-machina** [comment]
@pugak825, they don't need to do that. Since the payout is below what you can receive when you win, they will consistently profit in the long term. Do this test: run a bot on a real account and another on a demo account. The same bot, press play simultaneously and use a bot that enters the trades at specific times (like at the beginning of every minute). You will see that both bots will have the same profit/loss.

@alex-net007 O site da binary USA a API deles para fazer as trades, ler sua conta e os dados do mercado. O seu bot é um de MT5, o que não tem nada haver com a pergunta do autor

**Comment by Bkind254** [comment]
> 

I think if by switching markets every time I trade and trading two times a day for a profit of 0.5% per trade, staking 0.35 with a 10,000 Usd account, I can gain long-term consistency without blowing the account. I will be switching trade types sometimes from odd/even to over/under. I will survive right?

**Comment by binary-ex-machina** [comment]
@Bkind254, it's very likely to blow your account if you trade that way. Digits are random, so you can break your account today or in 1 year. Can you handle that doubt for one year? Another question is, when the bot is at the 12th loss in sequence, will you let the bot trade typically, or will you stop the bot manually? 12th loss in a row with starting stake of 0.35, and martingale 2.1 will give a stake of $1225

**Comment by Bkind254** [comment]
Thank you all
